src/cmd-prune

```diff
@@ -175,12 +175,13 @@ elif os.path.islink("builds/latest"):
 error_during_pruning = False
 for build in builds_to_delete:
     print(f"Pruning build {build.id}")
-    try:
-        subprocess.check_call(['ostree', '--repo=tmp/repo', 'refs', '--delete', build.id])
-        rmtree(os.path.join(builds_dir, build.id))
-    except Exception as e:
-        error_during_pruning = True
-        print(f"{e}")
+    if os.path.exists('tmp/repo'):
```

Comment:
Hmm, this is gating both the repo pruning and the build dir deletion. That seems unexpected. Let's make just the `ostree refs --delete` conditional?